tests/integration/routes/test_errors.py

```diff
@@ -1,9 +1,9 @@
-from unittest.mock import patch
+from mock import Mock, patch
```

Comment:
Worth updating the assert to cover for no cookie, the business content and the "other surveys" content.